src/value/mod.rs

```diff
@@ -258,6 +260,21 @@ pub trait Element {
     /// This will return `None` if the type is not `int` or the value is any `null`.
     fn as_any_int(&self) -> Option<&AnyInt>;
 
+    /// Returns a reference to the underlying float value for this element.
```

Comment:
Nit: the doc strings for `as_float` and `as_boolean` indicate that those methods will return `a reference to the underlying <<value>>`, but it's actually a copy.